doc/source/report/overview.rst

```diff
@@ -148,7 +148,38 @@ whole project can be divided into 6 sub-projects:
 -  **Strain:** A strain here is a genetic variant or sub-type of of the
    more general mouse population.
 
-5. References
+5. Data 
+-------
+The original data includes two folders, intervals and txy_coords, and a npy file named all_features_mousedays_11bins.
+The all_features_mousedays_11bins is a 9*1921*11 matrix, which represents 9 features among 1921 mouse days in 11 2 hour
+bins for a day, the 9 features are:
+
+-  **Food (F):** records the food consumption (g) for a certain mouse day and a certain time bin.
+-  **Water (W):** records the water consumption (g) for a certain mouse day and a certain time bin.
+-  **Distance (D):** records the movement distance for a certain mouse day and a certain time bin.
+-  **ASProbability (ASP):** records the AS time proportion in the certain time bin.
+-  **ASNumbers (ASN):** records the numbers of AS in the certain time bin. 
+-  **ASDurations (ASD):** records the total duration of AS in a certain bin.
+-  **ASFoodIntensity (ASFI):** equals F/ASP.
+-  **ASWaterIntensity (ASWI):** equals W/ASP. 
+-  **MoveASIntensity (ASMI):** equals D/ASP.
+
+The intervals folder has 6 sub-folders, all sub-folders have about 33 files for 3 strains and 11 days for each strain:
```

Comment:
line 167:  this line has some grammar issues, and it can be paraphrased as below: 
The intervals folder has 6 sub-folders. All these sub-folders have about 33 files for 3 strains, and for each strain there are 11 days data.